x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/transform/action/GetTransformStatsAction.java

```diff
@@ -53,6 +54,7 @@ public static final class Request extends BaseTasksRequest<Request> {
         private final String id;
         private PageParams pageParams = PageParams.defaultParams();
         private boolean allowNoMatch = true;
+        private boolean isBasicStats = false;
```

Comment:
Also removed `setIsBasicStats` since I was just following the pattern set by `allowNoMatch`, but there's no need to make this boolean mutable since it is only set on HTTP request and on transport stream out/in.